<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyMafia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 25, 25);</string>
  </property>
  <widget class="QLineEdit" name="Message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
   </property>
   <property name="placeholderText">
    <string>메세지를 입력해보세요!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MessageButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>650</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>메세지 전송</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MessageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="UserBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RoomInfoLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>방에 접속한 유저</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>방의 주소: | 닉네임: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MessageButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border: none;
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>방 퇴장</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
